### DISCUSSION TITLE
How do you get zigbee2mqtt running on MacOS?

### DISCUSSION THREAD

**Discussion by aatifk** [Koenkk/zigbee2mqtt#18700] [General]
I've been looking for a guide but there doesn't seem to be one. The official install instructions also don't cover MacOS. I understand that it's quite similar to Linux but what I'm struggling to understand is how to get the service to start/run automatically on system bootup. Is there anything that exists?

**Comment by iantait** [comment]
I use a plist:
[https://www.reddit.com/r/homebridge/comments/vbe17c/anyone_is_using_zigbee2mqtt_on_macos_can_you/](url)

**Reply by aatifk** [reply]
it says this page isn't available. Any chance you have another one?

**Comment by iantait** [comment]
You just need to google zigbee2mqtt plist

**Reply by aatifk** [reply]
Thank you, found it! I'll try to use this.
